src/analyzer.ts

```diff
@@ -261,7 +273,7 @@ export class Analyzer {
       warnings.push({
         code: 'could-not-load',
         message: `Unable to load import: ${error.message || error}`,
-        sourceRange: scannedImport.sourceRange,
+        sourceRange: scannedImport.urlSourceRange || scannedImport.sourceRange,
```

Comment:
I guess that's my question: when won't we have a URL, or some other kind of import identifier?